massa-execution-worker/src/interface_impl.rs

```diff
@@ -547,8 +562,8 @@ impl Interface for InterfaceImpl {
     /// * `to_address`: string representation of the address to which the coins are sent
     /// * `raw_amount`: raw representation (no decimal factor) of the amount of coins to transfer
     fn transfer_coins(&self, to_address: &str, raw_amount: u64) -> Result<()> {
-        let to_address = massa_models::address::Address::from_str(to_address)?;
-        let amount = massa_models::amount::Amount::from_raw(raw_amount);
+        let to_address = Address::from_str(to_address)?;
+        let amount = Amount::from_raw(raw_amount);
```

Comment:
Are we getting any problems from address deserialization/fromm stringing?

Reply:
I didn't modify anything here, just formatted the file